backend/src/services/auth/auth.service.ts

```diff
@@ -12,7 +12,7 @@ class Auth {
     this.#userRepository = userRepository;
   }
 
-  signUp(payload: UserCreatePayload): Promise<User> {
+  public signUp(payload: UserCreatePayload): Promise<User> {
```

Comment:
I would suggest not specifying a public access modifier. Since the absence of an access modifier in the typescript means that access to the class methods is public.